### PR TITLE
Editorial Workflow

### PR DESCRIPTION
**Pull request by bbengfort**
This PR adds the `blog` archetype and a `new-post.sh` script to easily create blogs and updates the README.md with the editorial workflow instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,115 @@
...
+
+```
+$ hugo new blog/YYYY-MM-DD-title-of-blog.md
+```
+
+To automatically add the date, you can use `hugo new blog/$(date +%F)-title.md` or you can use the helper script we've created as follows:
+
+```
+$ ./bin/new-blog.sh "Title of My Awesome Blog Post"
+```
```
Comment (rebeccabilbro) [inline README.md:45]: So convenient! Thank you for providing both options 🙂

File: README.md [hunk 2]
```diff
@@ -1,3 +1,115 @@
...
+
+```
+$ cwebp -q 80 static/images/test.jpg -o static/images/test.webp
+```
+
+The `-q` flag is a "quality flag" which should be a number between 0 (worst quality) and 100 (best quality). This flag controls the lossiness of the compression algorithm. Alternatively, you can use the `-lossless` flag to maximize the quality of the conversion. The `-o` flag is the output path to save the file to.
+
+## I18n and L10n
+
+[TODO: add details]
```
Comment (rebeccabilbro) [inline README.md:86]: Do we want to make this a "coming soon" and add a backlog issue to address this?

**Review by rebeccabilbro — APPROVED** [review]
Thank you so much @bbengfort for your thoughtful addition of our new editorial workflow &mdash; this all looks terrific! 

I really like the step-by-step instructions in the README, and that you are providing two different methods to create a new blog &mdash; one for folks who are used to using `hugo`, and one all-in-one-script. The datetime shortcuts are also a slick addition, including having the timezone in the blog template 😉 

My only suggestion is that we may want to add two backlog tasks for (1) addressing i18n/i11n and (2) adding an author page for Edwin under `content/en/author/`. Also, do we want to remove the directories `content/english/` and `content/french/` now that we have the much more convenient and consistently named `content/en/` and `content/fr/`?

Thank you again and very excited to get blogging!

File: README.md (hunk 2 above)
Comment (bbengfort) [reply README.md:86]: Good suggestion, I've done so!

**Comment by bbengfort** [comment]
@rebeccabilbro Thank you so much for your kind review! I've added the I18n task to the backlog and have checked, the `content/english` and `content/french` directories have been removed on my system; hopefully when you pull they will also be renamed locally for you - let me know if they are not.

**Comment by rebeccabilbro** [comment]
Sounds good! Actually those folders were not automatically removed (that's what prompted my earlier comment):

Screen Shot 2021-03-22 at 11 07 08 AM

But I have removed them manually from my system now, so no big deal.

**Comment by bbengfort** [comment]
@rebeccabilbro interesting - I wonder why that happened; there is nothing in them right? Actually, I just checked, apparently, this is a known bug with git: https://git-scm.com/docs/git-mv#_bugs

**Pull request merged**